common/spec/fixtures/github/labels_with_custom_and_default.json

```diff
@@ -0,0 +1,58 @@
+[
+  {
+      "id": 618335045,
+      "url": "https://api.github.com/repos/dependabot/dependabot-core/labels/bug",
+      "name": "bug",
+      "color": "ee0701",
+      "default": true
+  },
+  {
+      "id": 675690998,
+      "url": "https://api.github.com/repos/dependabot/dependabot-core/labels/dependencies",
+      "name": "Dependency: Gems",
```

Comment:
Should we add a lowercase `dependenci*` test fixture as well? I guess my concern is you could reverse the OR statement in `default_dependencies_label` and this test would pass if the match became case insensitive.

Reply:
Yeah good shout 👌